MekHQ/data/stratconcontractdefinitions/SecurityDuty.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ScenarioTemplate>
+    <alliedFacilityCount>0</alliedFacilityCount>
```

Comment:
Count of 0 when defending an allied facility?

Reply:
These are "additional" allied facilities on top of any spawned by objectives.